desktop/core/src/desktop/js/onePageViewModel.js

```diff
@@ -621,7 +621,8 @@ class OnePageViewModel {
       {
         url: '/filebrowser/download=*',
         app: function (ctx) {
-          location.href = window.HUE_BASE_URL + '/filebrowser/download=' + ctx.params[0];
+          const filePathParam = getModifiedCtxParamForFilePath(ctx, '/filebrowser/download=*');
+          location.href = window.HUE_BASE_URL + '/filebrowser/download=' + filePathParam;
```

Comment:
Nit: Maybe we can add `/filebrowser/download=` as a prefix and reuse everywhere?

Reply:
Like in `url.js` or `api.ts`

Reply:
Yes, but the drawback would be breaking existing consistency of having all of the URLs in the pageMapping object use string literals for readability.